.github/scripts/shared/label.ts

```diff
@@ -27,6 +27,12 @@ export const externalContributorLabel: Label = {
   description: 'Issue or PR created by user outside org',
 };
 
+export const needsTriageLabel: Label = {
+  name: 'needs-triage',
+  color: '#FA8072',
```

Comment:
Your PRs made me notice that the `needs-triage` didn't have the same color and description on both repos.
I just aligned both for consistency. Here's the color we can use:
```
color: '#68AEE6',
```